index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+  <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+</head>
+<body>
+  <section class="projects">
+    <article>
+      <h1>MY RECENT WORK</h1>
+    </article>
+
+    <hr>
+
+    <section class="ellipse_19">
+      <figure class="img_placeholder">
+        <img src="placeholder.png" alt="">
+      </figure>
+
+      <article>
+        <h1 class="title_post">Multi-Post Stories</h1>
```

Comment:
- [ ] Kindly change this tag to an `h2` tag in all the cards as this is the right semantic tag, otherwise great effort, the next reviewer should approve your work, all the best